cea/default.config

```diff
@@ -767,30 +767,30 @@ output-path = {general:scenario}/../..
 output-path.type = PathParameter
 output-path.help = Path to location where to store the new project
 
-[disconnected-cooling]
+[decentralized]
 AHUflag = false
 AHUflag.type = BooleanParameter
-AHUflag.help = this is a configuration flag, which influences the disconnected building calculations
+AHUflag.help = Flag for decentralized building calculations satisfying demand from only Air Handling
 
 ARUflag = false
 ARUflag.type = BooleanParameter
-ARUflag.help = this is a configuration flag, which influences the disconnected building calculations
+ARUflag.help = Flag for decentralized building calculations satisfying demand from only Air Recirculation
```

Comment:
What happens if AHUflag AND ARUflag are set?

Reply:
It will do the calculations separately, and it is run twice